ci-operator/jobs/Intersmash/intersmash/Intersmash-intersmash-main-periodics.yaml

```diff
@@ -0,0 +1,64 @@
+periodics:
+- agent: kubernetes
+  cluster: build01
```

Comment:
just noticed `context: ci/prow/intersmash-e2e` ... don't anything about its usage but `ci-operator/jobs/Intersmash/intersmash/Intersmash-intersmash-main-presubmits.yaml` has it

Reply:
Yep, that is part of `presubmit` job spec, specifically of the complex `report` property, see https://pkg.go.dev/k8s.io/test-infra/prow/config#Reporter and it is not part of the [periodic jobs spec](https://pkg.go.dev/k8s.io/test-infra/prow/config#Periodic). Therefore it must not be used here.